Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
Sure, go ahead, be all professional 'n' stuff :-)

Reply:
I swear by bundler these days.